install_config/topics/configuring_gce_objects.adoc

```diff
@@ -0,0 +1,176 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+[id='configuring-gce-objects_{context}']
+= Configuring Google Compute Engine objects
+
+Integrating {product-title} with Google Compute Engine (GCE) requires the
+following components or services.
+
+A GCP project::
+
+A GCP project is the base level organizing entity and forms the basis for
+creating, enabling, and using all GCP services. This includes managing APIs,
+enabling billing, adding and removing collaborators, and managing permissions.
+
+See link:https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects[The project resource section in the GCP documentation] for more information.
+
+[IMPORTANT]
+====
+Project IDs are a unique identifier, and project IDs must be unique across
+all of Google Cloud Engine. This means you cannot use `myproject` as a project
+ID if someone else has created a project with that ID before.
+====
+
+Billing::
+
+New resources cannot be created unless billing is attached to an account. The
+new project can be linked to an existing project or new information can be
+entered.
+
+See link:https://cloud.google.com/billing/docs/how-to/manage-billing-account[Create, Modify, or Close Your Billing Account in the GCP documentation] for more information.
+
+Cloud identity and access management::
+
+Deploying {product-title} requires the proper permissions. A user must be able
+to create service accounts, cloud storage, instances, images, templates, Cloud
+DNS entries, and deploy load balancers and health checks. Delete permissions are
+also helpful in order to be able to redeploy the environment while testing.
+
+You can create service accounts with specific permissions, then use them to
+deploy infrastructure components instead of regular users. You can also create
+roles to limit access to different users or service accounts.
+
+GCP instances use service accounts to allow applications to call GCP APIs. For
+example, {product-title} node hosts can call the GCP disk API to provide a
+persistent volume to an application.
+
+Access control to the various infrastructure, service resources, and
+fine-grained roles are available using the IAM service. For more information, see link:https://cloud.google.com/compute/docs/access/[the Access cloud overview section of the GCP documentation].
+
+SSH keys::
+
+GCP injects SSH public keys as authorized keys to be able to log in using SSH in
+the created instances. The list of SSH keys can be configured per instance or
+per project.
+
+You can use existing SSH keys. GCP metadata can help store the SSH keys that are
```

Comment:
This should be part of the previous paragraph.
"GCP metadata can help store..." Are you explicitly storing the keys in the metadata? Maybe "To allow SSH access to instances, store the SSH keys in the GCP metadata that is added to the instances at boot time."